lookout/style/format/visualizer/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+models/model.asdf
```

Comment:
why not all directory?

Reply:
only model.asdf is used so normally there is no reason to put any other file in it.